docs/pkg-tutorial.md

```diff
@@ -0,0 +1,259 @@
+# Packaging Tutorial
+
+This tutorial provides an end to end description of configuring and maintaining a uenv recipe and deployment for a representative HPC application.
+
+1. Gathering requirements
+2. Writing the recipe
+3. Testing
+4. Configuring the deployment
+5. CI/CD
```

Comment:
I was reading `pkg-application-tutorial.md` first and thought it was missing a target audience. This section covers it for me but depending on if the other file is needed, maybe that one also needs a short intro describing who it's aimed at?